rakefile.netstd.rb

```diff
@@ -0,0 +1,405 @@
+require 'albacore'
+
+msbuild_command = "C:/Program Files (x86)/MSBuild/14.0/Bin/MSBuild.exe"
+if !File.file?(msbuild_command)
+  raise "MSBuild not found"
+end
+
+nuget_command   = ".nuget/nuget.exe"
+gitlink_command = "packages/gitlink.2.3.0/lib/net45/GitLink.exe"
+xunit_command   = "packages/xunit.runner.console.2.0.0/tools/xunit.console.exe"
+
+solution        = "FakeItEasy.netstd.sln"
+assembly_info   = "src/CommonAssemblyInfo.cs"
+version         = IO.read(assembly_info)[/AssemblyInformationalVersion\("([^"]+)"\)/, 1]
+version_suffix  = ENV["VERSION_SUFFIX"]
+repo_url        = "https://github.com/FakeItEasy/FakeItEasy"
+nuspec          = "src/FakeItEasy.netstd/FakeItEasy.nuspec"
+analyzer_nuspec = "src/FakeItEasy.Analyzer/FakeItEasy.Analyzer.nuspec"
+logs            = "artifacts/logs"
+output          = File.absolute_path("artifacts/output")
+tests           = "artifacts/tests"
+packages        = File.absolute_path("packages")
+
+gitlinks        = ["FakeItEasy", "FakeItEasy.Analyzer", "FakeItEasy.netstd"]
+
+unit_tests = [
+  "tests/FakeItEasy.Tests/bin/Release/FakeItEasy.Tests.dll",
+  "tests/FakeItEasy.Analyzer.Tests/bin/Release/FakeItEasy.Analyzer.Tests.dll"
+]
+
+netstd_unit_test_directories = [
+  "tests/FakeItEasy.Tests.netstd"

```

Comment:
Or even more generally, given that the multi-targettting (project.json+xproj and future csproj) is "the future", those projects should probably have no suffix at all, and it should be the specific .NET Framework projects which should have a suffix, e.g. `net40`. I don't think we need to solve all this in this PR though. We can clear up the suffixes later.